### ISSUE TITLE
rename _master_ to _main_

### ISSUE THREAD

**Issue by mihai-sysbio** [MetabolicAtlas/standard-GEM#6]
There is a discussion on the web to rename the _master_ branch to _main_, for diversity reasons.
Somebody from GitHub (was it the CEO?) said something about potentially a GitHub-wide implementation. I'm not sure if it's best to wait for that rather than to push for the switch immediately.

**Comment by Hao-Chalmers** [comment]
IMHO, immediately

**Comment by Midnighter** [comment]
We renamed the master branch on cobrapy to stable. I agree, the sooner you do it the less hassle for everyone.

**Comment by Midnighter** [comment]
Please note that you should also adjust the standard document description:

> :red_square: Git branches
> The GEM repository must have at least two branches: master and develop.

**Comment by mihai-sysbio** [comment]
Thanks for the notice @Midnighter. As of the current version, it's still `master`.
This change will be implemented for the next release, unless we find reasons not to.

**Comment by draeger** [comment]
Please note that GitHub officially says to wait for its new updates which will seamlessly provide support for changing master branch to main branch.

Guidelines by GitHub on renaming branch name: https://github.com/github/renaming. At the moment, renaming it can cause problems when using GitHub pages.

**Comment by mihai-sysbio** [comment]
I was not aware of that, thanks for sharing that link @draeger!
Consequently, this issue is in the Backlog for now, to be dealt with after GitHub implements the changes. It sounds like a better idea to ask people to rename the branches only after that.

**Comment by edkerk** [comment]
Since January 2021, GitHub [no longer advises](https://github.com/github/renaming/commit/040636ea50e945875c70cc964d586f645e9bd810) to wait with changing `master` to `main`, so this can be implemented.

**Comment by mihai-sysbio** [comment]
This issue has been resolved via #32. Please open a new issue if any follow-up is required.
